### ISSUE TITLE
[EXTENSION] getting revolut data 

### ISSUE THREAD

**Issue by 0xChqrles** [keep-starknet-strange/zkramp#9]
Allow the extension sidebar to redirect the user to revolut. then extract the Revolut ID from the webpage to make it accessible for the zkRamp app

**Comment by BryanBorck** [comment]
can I work in this one?

**Comment by BryanBorck** [comment]
Yeah I think you can get it, I'm having some personal problems to finish this issue.

I already implemented some parts, if you want some help please tag me on telegram (same user there)

**Comment by Manush-2005** [comment]
Ok bro

**Comment by mateocovacho** [comment]
Hey @0xChqrles , I am interested in solving this issue
